flow360/component/simulation/models/surface_models.py

```diff
@@ -111,13 +100,26 @@ class SymmetryPlane(BoundaryBase):
     type: Literal["SymmetryPlane"] = pd.Field("SymmetryPlane", frozen=True)
 
 
+class Translational(Flow360BaseModel):
+    pass
+
+
+class Rotational(Flow360BaseModel):
+    axis_of_rotation: Optional[Axis] = pd.Field(None)
```

Comment:
Is this really optional?

Reply:
Yes. If user doesn't define we will try to automatically figure out the axis of rotation

Reply:
Ok then how does pydantic now which to construct when getting an empty dict? Maybe we need discriminator in this case? We will need to test this.